apps/dashboard/src/app/knowledge-box/knowledge-box-home/knowledge-box-home.component.ts

```diff
@@ -134,6 +134,6 @@ export class KnowledgeBoxHomeComponent implements OnInit, AfterViewInit {
   }
 
   downloadDesktop() {
-    window.open('https://github.com/nuclia/frontend/releases/tag/latest');
+    window.open('https://github.com/nuclia/frontend/releases');
```

Comment:
It seems this other url shows only the last version:
`https://github.com/nuclia/frontend/releases/latest`